api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -112,6 +113,26 @@ service StellarStationService {
   // `ListUpcomingAvailablePasses` should be called again to get a fresh list of reservable passes.
   // The pass must be scheduled at least one hour in advance of `aos_time` or it will be rejected.
   rpc ReservePass (ReservePassRequest) returns (ReservePassResponse);
+
+  // Add TLE for a satellite.
+  //
+  // If the TLE is accepted, it will be applied when executing upcoming plans, and for pass
+  // calculation..
+  //
+  // An 'INVALID_ARGUMENT' error will be returned in the following conditions:
+  // - the TLE cannot be parsed
+  // - the TLE is older than the current TLE
+  // - the TLE norad ID does not correspond to the requested satellite.
+  //
+  // If the satellite is not found or the client is not authorized for it, the request will return
+  // a `NOT_FOUND` error.
+  rpc AddTle (AddTleRequest) returns (AddTleResponse);
```

Comment:
I feel UpdateTle sounds better which gives an impression that TLE will be updated immediately for the satellite while AddTle sounds it is stored at somewhere in StellarStation, and it is not very clear when it is applied. Wha do you think?

Reply:
The problem with `UpdateTle` becomes when it sounds like the first TLE cannot come from the method. Can't come up with any better alternative to `AddTle`.

Reply:
How about provideTle? I don't think "Provide" is used very often as a verb in APIs, I heard the word when talking with external people and Ops team.

Reply:
In software APIs provide would tend to imply that the method returns (provides) something rather than having something given to it.

How about 'SetTle' or 'SetCurrentTle' or something like that?

The API only allows setting the most recent TLE so I guess these might be OK? (Internally we aren't actually replacing the TLE at all, but that's an implementation detail I guess).

Reply:
`SetTle` seems good

Reply:
SetTle seems good to me as well.